duckduckgo/Android
# Fix share failure

Task/Issue URL: https://app.asana.com/0/361428290920652/611385370877136
Tech Design URL: N/A

**Description**:
Removes the manual launching of share intents into tabs allowing live data updates to trigger them. This fixes an issue where sharing from an external app such as Chrome sometimes failed, resulting in a blank tab.

**Steps to test this PR**:
1. With the app open, share a url from chrome to our app and ensure the shared url displays
1. With the app closes, share a url from chrome to our app and ensure the shared url displays
1. Now repeat the above steps with developer settings set to "Don't keep activities" on and "Background process limit" set to one
1. Reset the settings changed in the previous step back to defaults

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
